src/IssueViz/Editor/LocationTagging/LocationTagger.cs

```diff
@@ -131,18 +148,25 @@ private void TranslateTagSpans(ITextSnapshot newSnapshot)
 
             foreach (var old in TagSpans)
             {
-                var newSpan = old.Span.TranslateTo(newSnapshot, SpanTrackingMode.EdgeExclusive);
-                var hasSpanBeenEdited = newSpan.Length != old.Span.Length;
-
-                if (hasSpanBeenEdited)
+                try
                 {
-                    // If the user has typed inside the tagged region we'll stop showing a Tag for that span
-                    old.Tag.Location.InvalidateSpan();
+                    var newSpan = old.Span.TranslateTo(newSnapshot, SpanTrackingMode.EdgeExclusive);
+                    var hasSpanBeenEdited = newSpan.Length != old.Span.Length;
+
+                    if (hasSpanBeenEdited)
+                    {
+                        // If the user has typed inside the tagged region we'll stop showing a Tag for that span
+                        old.Tag.Location.InvalidateSpan();
+                    }
+                    else
+                    {
+                        old.Tag.Location.Span = newSpan;
+                        translatedTagSpans.Add(new TagSpan<IIssueLocationTag>(newSpan, old.Tag));
+                    }
                 }
-                else
+                catch (Exception e)
```

Comment:
Wrapping with a try-catch just so we won't crash next time, in case there are other corner cases

Reply:
Don't suppress critical exceptions